manifests/init.pp

```diff
@@ -222,8 +222,14 @@
   String $jvm_args                                    = $rundeck::params::jvm_args,
   Hash $kerberos_realms                               = $rundeck::params::kerberos_realms,
   String $key_password                                = $rundeck::params::key_password,
-  Enum['db', 'file'] $key_storage_type                = $rundeck::params::key_storage_type,
+  Enum['db', 'file', 'vault'] $key_storage_type       = $rundeck::params::key_storage_type,
   Stdlib::Absolutepath $keystore                      = $rundeck::params::keystore,
+  String $vault_keystorage_prefix                     = $rundeck::params::vault_keystorage_prefix,
```

Comment:
For all those values, an empty string isn't valid, right? In that case, please use:
```
  String[1]
```